### ISSUE TITLE
List mode preset time is wrong by a factor of 10

### ISSUE THREAD

**Issue by MarkRivers** [epics-modules/Dante#13]
If the library function start_list() is called with a preset time of X seconds acquisition stops after X/10 seconds.

Here is the proof with the output of camonitor and debugging in the driver:

Preset time is 30 seconds:
```
Dante:dante1:PresetReal        2021-04-05 18:30:55.933592 30

2021/04/05 18:31:26.290 Calling start_list with time=30.000000
IDante:dante1:MCAAcquiring      2021-04-05 18:31:26.317731 Acquiring
Dante:dante1:MCAAcquiring      2021-04-05 18:31:29.331684 Done
```
So it only counted for 3 seconds rather than 30 seconds.

Preset time is 50 seconds:
```
Dante:dante1:PresetReal        2021-04-05 18:31:35.358914 50

2021/04/05 18:31:39.252 Calling start_list with time=50.000000
Dante:dante1:MCAAcquiring      2021-04-05 18:31:39.276767 Acquiring
Dante:dante1:MCAAcquiring      2021-04-05 18:31:44.296020 Done
```
So it only counted for 5 seconds rather than 50 seconds.

**Comment by lucagrittiniXGLAB** [comment]
Hi Mark,
this issue has been solved in the library I committed today (3.7.13.0)

**Comment by MarkRivers** [comment]
This problem was verified to be fixed in the new version of the library.
